packages/openactive-broker-microservice/app.js

```diff
@@ -90,54 +91,14 @@ if (REQUEST_LOGGING_ENABLED) {
   app.use(logger('dev'));
 }
 
-/**
- * Use OpenActive validator to validate the opportunity
- *
- * @param {any} data opportunity JSON-LD object
- */
-async function validateAndStoreValidationResults(data, validator) {
-  const id = data['@id'] || data.id;
-  const errors = await validator.validateItem(data, ITEM_VALIDATION_MODE);
-  if (!errors) return;
-  for (const error of errors) {
-    // Use the first line of the error message to uniquely identify it
-    const errorShortMessage = error.message.split('\n')[0];
-    const errorKey = `${error.path}: ${errorShortMessage}`;
-
-    // Ignore the error that a SessionSeries must have children as they haven't been combined yet.
-    // This is being done because I don't know if there is a validator.validationMode for this, and without ignoring the broker does not run
-    if (data['@type'] === 'SessionSeries' && errorShortMessage === 'A `SessionSeries` must have an `eventSchedule` or at least one `subEvent`.') {
-      // eslint-disable-next-line no-continue
-      continue;
-    }
-
-    // Create a new entry if this is a new error
-    let currentValidationResults = state.validationResults.get(errorKey);
-    if (!currentValidationResults) {
-      currentValidationResults = {
-        path: error.path,
-        message: errorShortMessage,
-        occurrences: 0,
-        examples: [],
-      };
-      state.validationResults.set(errorKey, currentValidationResults);
-    }
-
-    // Keep track of examples of each error, with a preference for newer ones (later in the feed)
-    currentValidationResults.occurrences += 1;
-    currentValidationResults.examples.unshift(id);
-    if (currentValidationResults.examples.length > 5) {
-      currentValidationResults.examples.pop();
-    }
-  }
```

Comment:
This logic is moved into ValidatorWorkerPool._processValidationError pretty much verbatim.

With the one exception that the filtering logic `if (data['@type'] === 'SessionSeries' && // ...` is moved into validator-worker's error filtering logic (`isIgnorableResult(..)`)